### ISSUE TITLE
Latest translations from Transifex missing "or"

### ISSUE THREAD

**Issue by jsha** [EFForg/https-everywhere#1778]
Steps to reproduce:

```
git clone https://git.torproject.org/translation.git
cd translation
git checkout https_everywhere_completed
cd ~/https-everywhere/src/chrome/locale/
for LOCALE in * ; do rsync -av ~/translation/$LOCALE/ $LOCALE/ ; done
cd ../../../
./test.sh
```

Expected result: Test passes.
Actual result: 
```
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
\_ missing for locale et
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
\_ missing for locale fo
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
ssl-observatory.prefs.show_cert_warning
ssl-observatory.prefs.show_cert_warning_tooltip
\_ missing for locale he
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
\_ missing for locale km
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
\_ missing for locale ko
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
ssl-observatory.prefs.show_cert_warning
ssl-observatory.prefs.show_cert_warning_tooltip
\_ missing for locale lt
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
ssl-observatory.prefs.show_cert_warning
ssl-observatory.prefs.show_cert_warning_tooltip
\_ missing for locale ms_MY
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
https-everywhere.menu.showCounter
https-everywhere.popup.keep
https-everywhere.popup.paragraph1
https-everywhere.popup.paragraph2
https-everywhere.popup.paragraph3
https-everywhere.popup.revert
https-everywhere.popup.title
https-everywhere.ruleset-tests.status_cancel_button
https-everywhere.ruleset-tests.status_start_button
https-everywhere.ruleset-tests.status_title
ssl-observatory.prefs.show_cert_warning
ssl-observatory.prefs.show_cert_warning_tooltip
\_ missing for locale si_LK
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
\_ missing for locale sk_SK
https-everywhere.about.or
https-everywhere.menu.blockHttpRequests
ssl-observatory.prefs.show_cert_warning
ssl-observatory.prefs.show_cert_warning_tooltip
\_ missing for locale sl
ERROR: Validation of locales failed.
```

**Comment by fuglede** [comment]
Hm, [I added](https://github.com/EFForg/https-everywhere/commit/82e83388365d51e7bcca59577fdfe8700da12915) that string; wasn't Transifex supposed to automatically be synced with those?

**Comment by jsha** [comment]
Copied from IRC:

18:29 < jsha> Transifex exports periodically to the "translations" repo on torproject: 
              https://gitweb.torproject.org/translation.git/tree/?h=https_everywhere_completed
18:30 < jsha> There's a different branch for each project
18:30 < jsha> Periodically, before doing a release, I copy over the contents of that repo, on that branch, into the HTTPS Everywhere repo
18:30 < jsha> You can see that Transifex's copy at 
              https://gitweb.torproject.org/translation.git/tree/si_LK/https-everywhere.dtd?h=https_everywhere_completed doesn't have the 'or' string.
18:30 < pred> Ah, I see.
18:30 < jsha> (I should really document this)
18:31 < pred> Sounds unpleasant for you at least. ;)
18:31 < jsha> (Also, I've been meaning to change this to a submodule)
18:31 < pred> What about syncing in the other direction then?
18:31 < jsha> It's not that bad. I think partly I still don't fully understand our translation system.
18:31 < jsha> I think I may have to manually go into Transifex and add "or" as a translatable string.
18:31 < jsha> Where I previously assumed it picked up strings automatically.

**Comment by fuglede** [comment]
Fixed by #2543, right?

**Comment by jsha** [comment]
Yep, fixed. Thanks!
